ecosystem/sep-0010.md

```diff
@@ -115,7 +119,10 @@ On success the endpoint must return `200 OK` HTTP status code and a JSON object
     * `manage_data(source: client_account, key: '<home domain> auth', value: random_nonce())`
       * The value of key is the home domain the challenge is authenticating, followed by `auth`. It can be at most 64 characters.
       * The value must be 64 bytes long. It contains a 48 byte cryptographic-quality random string encoded using base64 (for a total of 64 bytes after encoding).
-    * zero or more `manage_data(source: server_account, ...)` reserved for future use
+    * subsequent operations, order unimportant:
```

Comment:
```suggestion
    * subsequent operations must conform to one of the following patterns listed below. The order of these `manage_data` does not matter.
```

Reply:
The statement `must conform to one of the following patterns` reads to me as if the server must choose one dot point and only implement that, which isn't the intention. The server does not need to choose only one of the listed patterns, it can include any of the data that is listed, and the order with which it includes them in the transaction is not important.

It's unclear from your comment what is unclear about the current text that you're wanting to make clear. Could you elaborate so I understand the problem you're solving with the suggestion?

Reply:
It was really a matter of verbosity, we can keep it as-is but I felt the current language was a bit too terse.

Reply:
Agreed, it's a bit terse. I'm struggling to come up with more words that don't raise other questions so I'm going to leave it for now.